### PR TITLE
Normalize usages of directory separators.

### PR DESCRIPTION
**Pull request by Netmosfera**
Mostly, allowed me to run tests on Win.

### CODE REVIEW

File: src/Node/AbstractNode.php [hunk 1]
```diff
@@ -80,10 +80,14 @@ public function getId(): string
     public function getPath(): string
     {
         if ($this->path === null) {
-            if ($this->parent === null || $this->parent->getPath() === null || $this->parent->getPath() === false) {
+            if (
+                $this->parent === null ||
```
Comment (sebastianbergmann) [inline src/Node/AbstractNode.php]: Please do not mix reformatting like this with what the pull request is actually about.

File: tests/tests/BuilderTest.php [hunk 2]
```diff
@@ -168,45 +181,52 @@ public function testReducePaths($reducedPaths, $commonPath, $paths)
...
-            ],
-            [
+            ];
+
+            yield [
                 [
-                    'Money.php'          => [],
-                    'MoneyBag.php'       => [],
-                    'Cash.phar/Cash.php' => [],
+                    "Money.php"          => [],
```
Comment (sebastianbergmann) [inline tests/tests/BuilderTest.php]: Please dont replace `'` with `"` here.

Comment (Netmosfera) [reply tests/tests/BuilderTest.php]: Only that? I thought to maintain the style consistent in that method. The double quotes are not needed in many other places.

**Comment by sebastianbergmann** [comment]
Thanks!

**Pull request merged**